website_compassion/forms/match_partner_form.py

```diff
@@ -51,9 +50,6 @@ def after_partner_match(self, partner, new_partner, partner_vals,
                         'active': True,
                         'contact_id': False
                     })
-            if new_partner:
```

Comment:
I think this needs to be put back here (see my comment in compassion-modules PR)